prisma/schema.prisma

```diff
@@ -212,3 +213,17 @@ model Label {
 
   @@map("Labels")
 }
+
+model ScrapImages {
+  id          String       @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
```

Comment:
If you pull the latest feature/M2 I believe the schema files should be split into separate model files